build01/marvin-mk2.nix

```diff
@@ -0,0 +1,65 @@
+{ pkgs, lib, config, ... }:
+let
+  userLib = import ../users/lib.nix { inherit lib; };
+
+  sources = import ../nix/sources.nix;
+  marvin-mk2 = pkgs.python3.pkgs.buildPythonApplication rec {
+    pname = "marvin-mk2";
+    version = "rolling";
+    src = import sources.marvin-mk2 { };
+
+    propagatedBuildInputs = with pkgs.python3.pkgs; [
+      aiohttp
+      gidgethub
+    ];
+  };
+
+  marvin-mk2-runtimeenv = with pkgs; [
+    marvin-mk2
+  ];
+
+in
+{
+  users.users.marvin-mk2.packages = [
+  ];
+  users.groups.marvin-mk2 = { };
+  users.users.marvin-mk2 = {
+    useDefaultShell = true;
+    uid = userLib.mkUid "mmk2";
+    extraGroups = [ "marvin-mk2" ];
+  };
+  nix.trustedUsers = [
+    "marvin-mk2"
```

Comment:
I am curious why it needs to be a trusted user.

Reply:
It absolutely doesn't, just a leftover from copy & pasting r-ryantm's definition. In my defense, I did say its work in progress ;) Thanks for catching!